src/matlab2tikz.m

```diff
@@ -6329,6 +6327,14 @@ function printAll(m2t, env, fid)
     end
     opts = opts_append(opts, key, value);
 end
+function opts = opts_addOpts(m2t, opts, key, subOpts, sep)
+    % add a key={Opts} pair to an options array
+    if ~exist('sep','var') || ~ischar(sep)
+        sep = ',';
```

Comment:
I absolutely agree with you that `', '` is more readable (and I prefer that as well, although I also quite like the `sprintf(',\n')` syntax in some cases).
However, I'm a bit reluctant to change this since that will change the hashes right now. I'd rather keep it for another PR such that this PR remains purely:
 - refactoring, and
 - a bug fix
without any other side effects.